package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cyclonedx/cdxgen",
-  "version": "9.8.4",
+  "version": "9.8.5",
```

Comment:
Thank you. We need the same change in the lock file as well, so the easiest thing to do is to do `npm install` and commit the changes.